src/sentry/snuba/metrics/fields/base.py

```diff
@@ -195,11 +196,11 @@ def _get_entity_of_metric_mri(
         raise InvalidParams
 
     entity_keys_set: frozenset[EntityKey]
-    if use_case_id == UseCaseKey.PERFORMANCE:
+    if use_case_id == UseCaseID.TRANSACTIONS:
```

Comment:
```suggestion
    if use_case_id in [UseCaseID.TRANSACTIONS, UseCaseID.SPANS]:
```

Assuming `generic_metrics` are also used for spans.

Reply:
```suggestion
    METRIC_PATH_MAPPING[use_case_id] is UseCaseKey.PERFORMANCE
```

Or like you did in the other change but I would prefer to just be explicit about it and use the `in []`. What do you think?

Reply:
Thanks! Aa discussed offline, I added a function in use_case_id_registry.py to retrieve the UseCaseKey